Database/getting-started-with-MySQL-DBaaS.md

```diff
@@ -15,11 +15,13 @@ During the Limited Beta there is no production Service Level Agreement.
 
 #### Audience
 
-Currently, this article is to support customers in the Limited Beta program.  Additionally, these instructions are specific to provisioning service directly through the DBaas user interface.  
+Currently, this article is to support customers in the Limited Beta program.  Additionally, these instructions are specific to provisioning service directly through the DBaaS user interface.  
 
 ## Overview
 
-Our MySQL Database-as-a-Service limited beta provides instant access to a single MySQL-compatible database instance with SSL support, daily backups held locally for 7 days and basic monitoring.  The free beta is limited to a 1vCPU/1GB database instance with up to 100 MB storage and support for 100 concurrent connections in IL1 datacenter.
+Our MySQL Database-as-a-Service limited beta provides instant access to a single MySQL-compatible database instance with SSL support, daily backups held for 7 days and basic monitoring.  The free beta allows a user to choose from 4 different sized plans, as follows:
+
+![Plans](../images/dbaas-plans-beta.png)
```

Comment:
just my opinion but a table would be far better here for viewing (particuarly on mobile) than a screen cap of the plans.  

**Micro**|**Small**|**Medium**|**Large**
-----------|-----------|--------------|------------
CPUs: 1 Core<br>Memory: 1 GB<br>Storage: 1 GB<br>Connections: 100|CPUs: 1 Core<br>Memory: 2 GB<br>Storage: 1 GB<br>Connections: 1200|CPUs: 2 Core<br>Memory: 6 GB<br>Storage: 64 GB<br>Connections: 3600|CPUs: 4 Core<br>Memory: 16 GB<br>Storage: 256 GB<br>Connections: 9600


```
**Micro**|**Small**|**Medium**|**Large**
-----------|-----------|--------------|------------
CPUs: 1 Core<br>Memory: 1 GB<br>Storage: 1 GB<br>Connections: 100|CPUs: 1 Core<br>Memory: 2 GB<br>Storage: 1 GB<br>Connections: 1200|CPUs: 2 Core<br>Memory: 6 GB<br>Storage: 64 GB<br>Connections: 3600|CPUs: 4 Core<br>Memory: 16 GB<br>Storage: 256 GB<br>Connections: 9600
```